### PR TITLE
[BEAM-445] BigtableIO: upgrade to 0.9.1

### PR DESCRIPTION
**Pull request by dhalperi**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

* Use the uber jar
* Remove OS classifier mumbo jumbo
* Move common dependency versioning to root pom

### CODE REVIEW

**Comment by lukecwik** [comment]
Will wait till you merge pr/681 and then update this one to remove references to the OS detection plugin in archetypes as well.

**Comment by lukecwik** [comment]
R: @lukecwik

File: examples/java/pom.xml [hunk 1]
```diff
@@ -195,17 +195,6 @@
         </executions>
       </plugin>
     </plugins>
-
-    <extensions>
```
Comment (swegner) [inline examples/java/pom.xml]: hooray!

File: examples/java/pom.xml [hunk 2]
```diff
@@ -195,17 +195,6 @@
...
-      <!-- Use os-maven-plugin to initialize the "os.detected" properties for Bigtable
-           in beam-sdks-java-io-google-cloud-platform.
-           See: https://cloud.google.com/bigtable/docs/using-maven#encryption -->
-      <extension>
-        <groupId>kr.motd.maven</groupId>
-        <artifactId>os-maven-plugin</artifactId>
-        <version>${os-maven-plugin.version}</version>
```
Comment (swegner) [inline examples/java/pom.xml]: you can also remove `os-maven-plugin.version` property definition from the root pom.

**Comment by dhalperi** [comment]
Closing so that Travis and Jenkisn will stop triggering

**Comment by dhalperi** [comment]
Merged the current master in. Hoping that this will solve issues.

**Comment by lukecwik** [comment]
LGTM

**Pull request closed without merging**